x/tally/module.go

```diff
@@ -144,5 +145,13 @@ func (am AppModule) BeginBlock(_ context.Context) error {
 
 // EndBlock returns the end block logic for the tally module.
 func (am AppModule) EndBlock(ctx context.Context) error {
+	start := telemetry.Now()
+	defer telemetry.ModuleMeasureSince(types.ModuleName, start, telemetry.MetricKeyEndBlocker)
```

Comment:
IMO The default leaves for very unclear metric names. I added my thoughts on why to the earlier conversation.